scripts/policy/protocols/conn/known-services.zeek

```diff
@@ -255,11 +257,11 @@ event successful_connection_remove(c: connection) &priority=-5
 	if ( c$known_services_done )
 		return;
 
-	if ( c$resp$state != TCP_ESTABLISHED && c$resp$state != UDP_ACTIVE )
-		return;
```

Comment:
Good points on UDP and ICMP, I haven't gone too far down that route.  From an operational standpoint, I think requiring the UDP server to respond makes sense, otherwise UDP scanning will just light it up.  ICMP I generally don't care about, but could see an option for people that do.